docs/en/stack/security/authorization/privileges.asciidoc

```diff
@@ -29,6 +29,11 @@ This includes snapshotting, updating settings, and rerouting. It also includes
 obtaining snapshot and restore status. This privilege does not include the 
 ability to manage security.
 
+`manage_ccr`::
+All {ccr} operations related to managing follower indices and auto-follow 
```

Comment:
I would highlight that this cluster level privilege only needs to be configured in the follower cluster.

Reply:
Thanks, I've added that info too!